shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/base/BaseITCase.java

```diff
@@ -358,6 +358,7 @@ protected void waitMigrationFinished(final String jobId) {
     }
     
     protected void assertGreaterThanInitTableInitRows(final int tableInitRows, final String schema) {
+        proxyExecuteWithLog("REFRESH TABLE METADATA", 2);
         String countSQL = StringUtils.isBlank(schema) ? "SELECT COUNT(*) as count FROM t_order" : String.format("SELECT COUNT(*) as count FROM %s.t_order", schema);
```

Comment:
Could we put `REFRESH TABLE METADATA` after `CHECK MIGRATION => STOP MIGRATION => CLEAN MIGRATION`?